aptos-move/e2e-move-tests/src/tests/remote_state.rs

```diff
@@ -0,0 +1,141 @@
+    bcs::from_bytes::<u64>(bytes.as_slice()).unwrap()
+}
+
+/// Simple test that reads the APT balance of the test account.
+#[ignore]
+#[tokio::test(flavor = "multi_thread")]
+async fn view_existing_account_balance() {
+    let mut h = MoveHarness::new_with_remote_state(AptosBaseUrl::Testnet, TEST_TXN_VERSION);
+
+    let existing_account_addr = AccountAddress::from_hex_literal(TEST_ACCOUNT_ADDR).unwrap();
+
+    assert_eq!(
+        get_account_apt_balance(&mut h, existing_account_addr),
+        TEST_ACCOUNT_APT_BALANCE
+    )
+}
+
+/// Transfers 1 APT from a newly created account to the existing test account
+/// and verifies that the recipient's balance increases accordingly.
+#[ignore]
+#[tokio::test(flavor = "multi_thread")]
+async fn transfer_to_existing_account() {
+    let mut h = MoveHarness::new_with_remote_state(AptosBaseUrl::Testnet, TEST_TXN_VERSION);
+
+    let existing_account_addr = AccountAddress::from_hex_literal(TEST_ACCOUNT_ADDR).unwrap();
+
+    // Create a new account and fund it with 10 APT.
+    let new_account =
+        h.new_account_with_balance_and_sequence_number(10_0000_0000 /* 10 APT */, 0);
+
+    // Transfer 1 APT to the existing account.
+    let status = h.run_entry_function(
+        &new_account,
+        str::parse("0x1::coin::transfer").unwrap(),
+        vec![TypeTag::from_str("0x1::aptos_coin::AptosCoin").unwrap()],
+        vec![
+            MoveValue::Address(existing_account_addr)
+                .simple_serialize()
+                .unwrap(),
+            MoveValue::U64(1_0000_0000).simple_serialize().unwrap(),
+        ],
+    );
+    assert_success!(status);
+
+    // Verify that the recipient's balance has increased by 1 APT.
+    assert_eq!(
+        get_account_apt_balance(&mut h, existing_account_addr),
+        TEST_ACCOUNT_APT_BALANCE + 1_0000_0000
+    )
+}
+
+/// Attempts to transfer 1 APT from the existing test account to a newly created account,
+/// verifying that the sender's balance decreases appropriately.
+#[ignore]
+#[tokio::test(flavor = "multi_thread")]
+async fn transfer_from_existing_account() {
+    let mut h = MoveHarness::new_with_remote_state(AptosBaseUrl::Testnet, TEST_TXN_VERSION);
+
+    let existing_account_addr = AccountAddress::from_hex_literal(TEST_ACCOUNT_ADDR).unwrap();
+
+    // Create a new account.
+    let new_account =
+        h.new_account_with_balance_and_sequence_number(10_0000_0000 /* 10 APT */, 0);
+
+    // Rotate the authentication key of the existing account so that we can authenticate transactions
+    // without using or exposing the real private key.
+    //
+    // This is a recommended security practice to prevent accidental leakage of the original private key,
+    // such as pushing to github accidentally.
+    let existing_account = h
+        .executor
+        .rotate_account_authentication_key(existing_account_addr);
+
+    // Transfer 1 APT from the existing account to the new account.
+    let status = h.run_entry_function(
```

Comment:
nit: can use gas_used from output, but does not matter too much if this adds complexity